docker-images/gwells-python-test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openshift/python:3.5
+FROM openshift/python:3.6
```

Comment:
do we still need this file?

Reply:
I don't think so.  Let's do some cleanup before wrapping this task.